### ISSUE TITLE
style RedundantExplicitType - not work

### ISSUE THREAD

**Issue by Proskurkin** [arturbosch/detekt#2039]
## Expected Behavior
The rule will work and the detekt will report an error.

## Observed Behavior
Detekt does not report an error.

## Steps to Reproduce
```
  RedundantExplicitType:
    active: true
```

```
fun str() {
    val x: String = "string"
}
```

I run Detekt plugin 

## Your Environment
Java 1.8.0_221
kotlin 1.3.41
detekt 1.1.1
Gradle 5.5.1
macOS

**Comment by arturbosch** [comment]
Same as #2038
